package.json

```diff
@@ -43,8 +43,8 @@
     "localforage": "1.7.3",
     "localforage-observable": "2.0.0",
     "mark.js": "8.11.1",
-    "marked": "github:mattermost/marked#3a8d6ffc804de8ecd6b749ef8ebc9a4a94067b80",
-    "mattermost-redux": "github:mattermost/mattermost-redux#e40072b407f8d437cb10a2edc78ac7972c7587be",
+    "marked": "github:mattermost/marked#09e4643601bc620c682d16410ddc4a4115815a12",
```

Comment:
Unintentional reversion of marked version?

Reply:
Hmm not sure what happened here!